src/main/java/domain/AttendanceStatus.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import java.time.DayOfWeek;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+public enum AttendanceStatus {
+    ATTENDANCE("출석"),
+    LATE("지각"),
+    ABSENCE("결석"),
+    TRUANCY("결석"),
+    ;
+
+    private final String expression;
+
+    AttendanceStatus(String expression) {
+        this.expression = expression;
+    }
+
+    public String getExpression() {
+        return expression;
+    }
+
+    public static AttendanceStatus of(LocalDateTime inputTime) {
+        DayOfWeek dayOfWeek = inputTime.getDayOfWeek();
+        int startHour = 10;
+        if (dayOfWeek == DayOfWeek.MONDAY) {
+            startHour = 13;
+        }
```

Comment:
startHour를 초기화하는 로직이네요. 여기는 아래와 같이 메서드 추출을 해보면 어떨까요?

```suggestion
        int startHour = getStartHour(dayOfWeek)
```

startHour를 구하는 로직과 AttendanceStatus를 구하는 로직이 혼합되어 있는 것 같습니다.

Reply:
[refactor(AttendanceStatus): 교육 시작 시간 계산 메서드 분리](https://github.com/woowacourse/java-attendance/pull/42/commits/bc8debd78c7c8cc7e595da07056e448403d6ac46)